sufia-models/app/jobs/copy_permissions_job.rb

```diff
@@ -0,0 +1,23 @@
+class CopyPermissionsJob 
+  def queue_name
+    :permissions
+  end
+
+  attr_accessor :id
+
+  def initialize(id)
+    self.id = id
+  end
+
+  def run
+    work = ActiveFedora::Base.load_instance_from_solr(id)
+    if work.respond_to?(:generic_files)
+      work.generic_files.each do |file|
+        file.visibility = work.visibility
+        file.edit_users = work.edit_users
```

Comment:
@scherztc I think this would be a better way to copy permissions without missing things

 `work.permissions.each {|perm| file.permissions << Hydra::AccessControls::Permission.new(perm.to_hash)}`